solana-programs/anchor/audius-data/tests/follows.ts

```diff
@@ -222,8 +225,64 @@ describe("follows", function () {
       expect(decodedData.followeeHandle.seed).to.deep.equal(
         constants2.handleBytesArray
       );
-      expect(accountPubKeys[0]).to.equal(adminStorageKeypair.publicKey.toString());
-      expect(accountPubKeys[3]).to.equal(newUser1Key.publicKey.toString());
+      expect(accountPubKeys[0]).to.equal(
+        adminStorageKeypair.publicKey.toString()
+      );
+      expect(accountPubKeys[5]).to.equal(newUser1Key.publicKey.toString());
+    });
+
+    it("delegate follows user", async function () {
+      const delegate = await testCreateUserDelegate({
```

Comment:
this is kind of a nit but instead of `delegate` shouldn't this be `user`? bc you get user info bundled with a test delegate account